<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: #f3f3f3;
border-radius: 40px 40px 40px 40px;
padding: 20px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>12</number>
        </property>
        <item>
         <widget class="QPushButton" name="newButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/new.svg</normaloff>:/icons/icons/new.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/open.svg</normaloff>:/icons/icons/open.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/save.svg</normaloff>:/icons/icons/save.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exportButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/export.svg</normaloff>:/icons/icons/export.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backgroundButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/background.svg</normaloff>:/icons/icons/background.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="brushLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Brush
0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="brushSlider">
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #000;
    height: 2px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #000;
    margin: 0px 0;
}

QSlider::handle:horizontal {
    background: #000;
    border: 0px solid #5c5c5c;
    width: 22px;
    height: 22px;
    margin: -10px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 11px;
}</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="eraserLabel">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Eraser
0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="eraserSlider">
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: 0px solid #000;
    height: 2px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #000;
    margin: 0px 0;
}

QSlider::handle:horizontal {
    background: #000;
    border: 0px solid #5c5c5c;
    width: 22px;
    height: 22px;
    margin: -10px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
    border-radius: 11px;
}</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="ColorPicker" name="colorPicker" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50000</width>
          <height>50000</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="firstButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/first.svg</normaloff>:/icons/icons/first.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previousButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/previous.svg</normaloff>:/icons/icons/previous.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="erasePageButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/erase.svg</normaloff>:/icons/icons/erase.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/next.svg</normaloff>:/icons/icons/next.svg</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="lastButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    border: 0px;
    border-radius: 6px;
	padding: 10px;
    background-color: #00000000;
}

QPushButton:hover {
    background-color: #33000000;
}

QPushButton:pressed {
    background-color: #66000000;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/icons/icons/last.svg</normaloff>:/icons/icons/last.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="pageCounterLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>1/2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorPicker</class>
   <extends>QWidget</extends>
   <header>colorpicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
